content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -451,17 +449,7 @@ avoid unnecessary retries within an index.
 
 ### Pod failure policy {#pod-failure-policy}
 
-{{< feature-state for_k8s_version="v1.26" state="beta" >}}
-
-{{< note >}}
-You can only configure a Pod failure policy for a Job if you have the
-`JobPodFailurePolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-enabled in your cluster. Additionally, it is recommended
-to enable the `PodDisruptionConditions` feature gate in order to be able to detect and handle
-Pod disruption conditions in the Pod failure policy (see also:
-[Pod disruption conditions](/docs/concepts/workloads/pods/disruptions#pod-disruption-conditions)).
-Both feature gates are available in Kubernetes {{< skew currentVersion >}}.
-{{< /note >}}
+{{< feature-state for_k8s_version="v1.31" state="stable" >}}
```

Comment:
Improvement for 'feature-state' shortcode usage here, consider using the new feature_gate_name option, making the feature state more data-driven.
```suggestion
{{< feature-state feature_gate_name="JobPodFailurePolicy" >}}
```

Reply:
Thanks, done